src/Stratis.Bitcoin.Features.Wallet/Controllers/WalletController.cs

```diff
@@ -1420,19 +1420,20 @@ public IActionResult RemoveTransactions([FromQuery]RemoveTransactionsModel reque
                 }
 
                 // If the user chose to resync the wallet after removing transactions.
-                if (result.Any() && request.ReSync)
```

Comment:
By removing the `result.Any()` here, are we implying that we will always re-sync regardless if any transactions was affected?

Reply:
Hm, I guess this is debatable. My feeling is that if you've specified a resync, it should happen, even if no changes land up being made.